src/Cake.Issues/FileLinkSettings.cs

```diff
@@ -81,5 +103,38 @@ public FileLinkSettings(string fileLinkPattern)
                     "&lineStartColumn={Column}" +
                     "&lineEndColumn={EndColumn}");
         }
+
+        /// <summary>
+        /// Returns settings for linking to files hosted in Azure DevOps for a specific commit id.
+        /// </summary>
+        /// <param name="repositoryUrl">Full URL of the Git repository,
+        /// e.g. <code>https://dev.azure.com/myorganization/_git/myrepo</code>.</param>
+        /// <param name="commitId">The commit id on which the file linking will be based on.</param>
+        /// <param name="rootPath">Root path of the files.
+        /// <c>null</c> or <see cref="string.Empty"/> if files are in the root of the repository.</param>
+        /// <returns>Settings for linking to files hosted in Azure DevOps.</returns>
+        public static FileLinkSettings AzureDevOpsCommit(
+            Uri repositoryUrl,
+            string commitId,
+            string rootPath)
+        {
+            repositoryUrl.NotNull(nameof(repositoryUrl));
+            commitId.NotNullOrWhiteSpace(nameof(commitId));
+
+            if (!string.IsNullOrWhiteSpace(rootPath))
+            {
+                rootPath = rootPath.Trim('/') + "/";
+            }
+
+            return
+                new FileLinkSettings(
+                    repositoryUrl.ToString().TrimEnd('/') +
+                    "?path=" + rootPath + "{FilePath}" +
+                    "&version=GC" + commitId +
+                    "&line={Line}" +
+                    "&lineEnd={EndLine}" +
+                    "&lineStartColumn={Column}" +
+                    "&lineEndColumn={EndColumn}");
```

Comment:
If `EndLine`, `Column` or `EndColumn` is not available, link won't highlight the line. This will taken care of later in a rewrite of the `FileLinkSettings` class